cmd/migrate/gear/core/1536736169_initialize_schema.up.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+CREATE TABLE _sky_user (
+  id text PRIMARY KEY,
+  token_valid_since timestamp without time zone,
+  last_seen_at timestamp without time zone,
+  disabled boolean NOT NULL DEFAULT false,
+  disabled_message text,
+  disabled_expiry timestamp without time zone
+);
```

Comment:
This table would be managed by core library.

`AuthInfoStore` in core library would provide `AuthInfo` crud.

Reply:
How about the public end-point of authinfo crud, likd disable user. I assume it will host at auth gear. So it will be auth gear responsible for http level, and DB level is by core? Am I correct?

Reply:
> it will be auth gear responsible for http level, and DB level is by core

yes